config/tech-docs.yml

```diff
@@ -4,7 +4,7 @@ host: https://ministryofjustice.github.io/operations-engineering
 # Header-related options
 show_govuk_logo: true
 service_name: MoJ Operations Engineering
-service_link: /operations-engineering
```

Comment:
This is the link used in the header bar, which needs to remain as `/operations-engineering` as the GitHub Pages URL is `https://ministryofjustice.github.io/operations-engineering` (not `https://ministryofjustice.github.io/`).

Reply:
It does mean the link doesn't work _locally_, however.

Reply:
ah, I see! Will revert :)